src/PureScript/Backend/Chez/Convert.purs

```diff
@@ -159,8 +159,9 @@ codegenExpr codegenEnv@{ currentModule } (NeutralExpr s) = case s of
 
   Accessor _ (GetProp _) ->
     S.Identifier $ "accessor-get-prop"
-  Accessor _ (GetIndex _) ->
-    S.Identifier $ "accessor-get-index"
+  Accessor e (GetIndex i) ->
+    S.List
+      [ S.Identifier $ scmPrefixed "vector-ref", (codegenExpr codegenEnv e), S.Identifier $ show i ]
```

Comment:
> Maybe you can use Syntax.chezUncurriedApplication instead of building the form by hand?

That is a bit simpler

> Why is the index translated to an Identifier and not to an Integer?

I'm so used to seeing `S.List` and `S.Identifier` that I forgot there were other things :laughing: